styles/output/nursing-internal-pdf.css

```diff
@@ -1805,104 +1805,79 @@ h4.os-subtitle + .os-figure:not(.has-splash) > figure {
   margin-left: auto;
   margin-right: auto;
 }
-
 .os-table:not(.os-unstyled-container) > table {
   border-collapse: collapse;
   width: 100%;
   margin-bottom: 0;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th {
-  border-bottom: 0.1rem solid black;
-  border-top: 0.1rem solid black;
+  border-bottom: 0.1rem solid #000000;
+  border-top: 0.1rem solid #000000;
   font-family: Mulish, sans-serif;
   font-weight: 700;
   padding: 0.7rem;
 }
-
-.os-table:not(.os-unstyled-container) > table > thead > tr > th[data-valign=top] {
```

Comment:
the updated selector here will no longer select an element ... if it was `> table th` it might be ok, but `> table > th` won't exist i think ...